samples/templates/default-azuredeploy-docker.json

```
@@ -662,8 +683,10 @@
             "apiVersion": "2019-06-01",
             "location": "[resourceGroup().location]",
             "properties": {
-                "supportsHttpsTrafficOnly": true
-            },
+                "supportsHttpsTrafficOnly": true,
+                "allowBlobPublicAccess": false,
+                "allowSharedKeyAccess": false
```

Lets move `allowSharedKeyAccess` to a different PR so we can get this one in. While the FHIR Server is fine with it the test pipeline is connecting using a key which is blocking us here.

```suggestion
                "supportsHttpsTrafficOnly": true,
                "allowBlobPublicAccess": false
```